setup.cfg

```diff
@@ -12,12 +12,11 @@ tag_prefix =
 parentdir_prefix = modin-
 
 [tool:pytest]
-addopts = --cov-config=setup.cfg --cov=modin --cov-append --cov-report=
+addopts = --cov-config=setup.cfg --cov=modin --cov-append --cov-report= -m "not exclude_by_default"
 xfail_strict=true
 markers =
-    xfail_executions
-    skip_executions
     exclude_in_sanity
+    exclude_by_default
```

Comment:
I expect this to be used in CI like we use `exclude_in_sanity`. Why do we not use it?

Reply:
`exclude_in_sanity` option disables tests only for CI, but does not disable them if you run them locally. This behavior does not correspond to `exclude_by_default` option, which disables tests everywhere by default.

Personally, I would not like to run these tests locally, as far as I remember they took a very long time to run.